web/web-portlet/src/java/org/sakaiproject/portlets/PortletIFrame.java

```diff
@@ -596,6 +596,11 @@ else if (SPECIAL_WORKSITE.equals(special))
 					    String infoUrl = StringUtils.trimToNull(s.getInfoUrl());
 					    if (infoUrl != null)
 					    {
+                                            //Check if infoUrl is relative? and prepend the server url
+                                            String serverUrl = ServerConfigurationService.getServerUrl();
+                                            if(infoUrl.startsWith("/") && infoUrl.indexOf("://") == -1){
+                                                infoUrl = serverUrl + infoUrl;
```

Comment:
Seems like we would always be calling SCS.getServerUrl even when we don't need to, try something like:
infoUrl = ServerConfigurationService.getServerUrl() + infoUrl;